src/core/tracking.js

```diff
@@ -98,6 +134,44 @@ export class Tracking {
       // (addons-frontend vs addons-server) is being used in analytics.
       ga('set', 'dimension3', 'addons-frontend');
     }
+
+    // Attempt to enable Hybrid content telemetry.
+    this.hctInitPromise = this.initHCT({ _config });
+  }
+
+  async initHCT({ _config = config } = {}) {
+    const hctEnabled = _config.get('hctEnabled');
+    if (typeof window !== 'undefined' && hctEnabled === true) {
+      log.info('Setting up the Hybrid Content Telemetry lib');
+      // Note: special webpack comments must be after the module name or
+      // babel-plugin-dynamic-import-node will blow-up.
+      const hybridContentTelemetry = await import('mozilla-hybrid-content-telemetry/HybridContentTelemetry-lib');
```

Comment:
The `await` should be used inside a `try/catch` block to avoid a potential issue with unhandled promises.